docs/graphql/manual/auth/authorization/role-multiple-rules.rst

```diff
@@ -0,0 +1,119 @@
+Multiple column + row permissions for the same role
+===================================================
+
+.. contents:: Table of contents
+  :backlinks: none
+  :depth: 1
+  :local:
+
+Use case
+--------
+
+In some cases we might want to allow access to certain columns for a role only if a particular condition is met
+while allowing access to other columns based on a different condition.
+i.e. have different column permissions based on different row permissions.
+
+Currently it is not possible to define multiple column + row permission rules for the same role.
+
+We can work around this limitation by using :ref:`views <custom_views>`.
+
+**For example**
+
+Let's say we have a table called ``user_info`` with columns ``(id, name, city, email, phone, address)``.
+
+We want the role ``user`` to be able to access:
+
+- the ``email``, ``phone`` and ``address`` columns only if the ``id`` column is the requesting user's id, i.e. current
+  user is owner of the row.
+
+- the ``id``, ``name`` and ``city`` columns for all rows.
+
+We can achieve this via the following steps:
+
+Step 1: Create a view 
+---------------------
+
+:ref:`Create <create_views>` a view ``user_private`` with columns ``(user_id, email, phone, address)``:
+
+.. code-block:: SQL
+
+    CREATE VIEW user_private AS
+      SELECT id AS user_id, email, phone, address
+        FROM user_info;
+
+Step 2: Create a relationship
+-----------------------------
+
+For the table ``user_info``, :ref:`create <create_manual_relationships>` a manual object relationship called
```

Comment:
Can we do the link on the word "manual object relationship" instead of "create"?